src/components/header.tsx

```diff
@@ -8,7 +8,8 @@ import { Locale } from "../pages";
 const isDemoSite = process.env.GATSBY_DEMO_SITE === "1";
 
 const TENANT_PLATFORM_URL =
-  "https://" + (isDemoSite ? "demo" : "app") + ".justfix.nyc/login";
+  (process.env.GATSBY_TENANT_PLATFORM_SITE_ORIGIN ||
+    "https://demo.justfix.nyc") + "/login";
```

Comment:
Hmm, so I'm not a huuuge fan of how the default value here is being repeated twice--I think it's fine since we're just using it twice, but once we have three repetitions (e.g. if we start integrating w/ the TP using another endpoint/URL) then I think we ought to consider bustin' up a simple module, e.g. `config.ts`, that just exports a `TENANT_PLATFORM_SITE_ORIGIN` which centralizes this default to keep things nice and DRY!  You're welcome to do that in this PR or we can save it for later if/when we add another reference to the origin.

Reply:
Right right that makes sense. I think your approach of waiting until the third use case of it makes sense. I'll file an issue for it so we can save it for that.

Reply:
Ok this is now being saved in #113